chevah_build

```diff
@@ -853,7 +853,7 @@ command_build_python_extra_libraries() {
         execute $PYTHON_BIN -m pip install $PIP_ARGS $library
     done
 
-    if [ $OS == "win" ]; then
+    if [ "$OS" = "win" -a "$ARCH" = "x86" ]; then
```

Comment:
this needs to be reverted as we need to patch on any ARCH and make sure our python has no external dependecies

Reply:
OK, will return with an updated version that uses own redistributables on X64 as well.

**needs-changes**